### ISSUE TITLE
Graphics: Stylization

### ISSUE THREAD

**Issue by petrSchreiber** [petrSchreiber/BetweenTheBubbles-tb#47]
Hi Sebastian,

this thread can help us discuss better the visual stylization of the game. Thanks to big help from my girlfriend, we have a first mockup of the "realistic" style.

Please, let me know if you would like to add some elements to it, once we agree on fact it is +/- representing possible gameplay screenshot, a stylized version will be done.

I have some concerns and ideas about the realistic one, but I am curious to hear about your inputs first.

**Comment by petrSchreiber** [comment]
![Náhled](https://user-images.githubusercontent.com/8292220/82747590-c858ab00-9d9a-11ea-9618-e801161c1e2c.png)

**Comment by DirectuX** [comment]
> I have some concerns and ideas about the realistic one, but I am curious to hear about your inputs first.

😄

> Thanks to big help from my girlfriend, we have a first mockup of the "realistic" style.

Thanks too 🙏

> Please, let me know if you would like to add some elements to it, once we agree on fact it is +/- representing possible gameplay screenshot, a stylized version will be done.

First I want to say that I still don't have a clear idea of the in-game view / FoV / viewpoint. i.e. is the camera fixed ? You talked about zooming too.  

I understand here that you want a sample of each object class to see differences between styles in future mockups; the mockup could be overloaded but I think it's ok, isn't it ?

**What I can already think of :** 

- > banks (hard obstacle of large size)
--> we need to design the banks in a way they don't give player idea you can climb from water, maybe some sharper "cliffs"

I can foresee a bit of bank at the upper left angle of the mockup. (see also picture in issue #33) ; with maybe a go-over-the-water leaf  ?

- > hard obstacle (can pose hit danger, in general a place you cannot go to)
--> for example stone, branch sticking out of the water

I can foresee both stone and branch as these are very different in look, all the more if the same picture will undergo different stylizing process.

- ( ⚠️ maybe not in your demands) camera (picture angle) tilt (see also first question at the bottom of this very comment). This will also allow better visualization of the bank environment.

- > flow direction area
--> some "debris" to visualize the flow would be nice

- a pickable item or power-up , or is it the round coin in the sand ?

- representation of health #22

- [!important] a stunned foe [Story#10](https://github.com/petrSchreiber/BetweenTheBubbles-secrets/issues/10#issuecomment-631563398)

<hr>

- We already have exchanged about this subject, but, as the outcome of the latests developments, how do you expect the world to be constructed, programmatically speaking ? Is it still 2D in 3D ? (If it is , please, can you rephrase this better ?) 

- Have you thoughts linked to [Story#11](https://github.com/petrSchreiber/BetweenTheBubbles-secrets/issues/11#issuecomment-631561024), especially upon the last comment ?

- Have you thoughts linked to [Story#9](https://github.com/petrSchreiber/BetweenTheBubbles-secrets/issues/9#issue-619761855) ?

**Please, feel free to ask clarification on any point.**

**Comment by petrSchreiber** [comment]
Sebastian,

my apologies for delay in reply - very intensive week. My apologies ahead I am reacting slowly and per partes. I am super hyped for Between the Bubbles, just need to handle properly workload to prepare more and more time for the Pilgrim :)

All your input will be incorporated (step by step), to the next iterations, coming soon 👍 Already discussed it with our mockup helper - Evža.

- for FOV / work with camera, I will prepare small POC this weekend for simple top-down
- I can see the game as mostly working in 2D (and flipped 2D), but with some 3D objects - will visualize

**Comment by DirectuX** [comment]
No worries, Petr, I understand. By the time I already started rewriting Cornelius, now Jack.

**Comment by DirectuX** [comment]
> All your input will be incorporated (step by step), to the next iterations, coming soon.

You did say you have concerns, about the realistic styling. Do you want to share them ? If we agree at the moment, it'll could spare time we can, for sure, redirect on another idea.

**Comment by DirectuX** [comment]
> I can see the game as mostly working in 2D (and flipped 2D), but with some 3D objects - will visualize

Thank you, visualization is the Key !

**Comment by petrSchreiber** [comment]
Hi Sebastian,

my concerns regarding realistic style are:
- it is very demanding for art - we can hit the issue of uncanny valley
- it can be harder to distinguish elements in the scene (high frequency visuals)
- it can result in a lot of high volume assets (game can grow big in size)

These are not blockers, but... a list of possible challenges :)

For example - hard to distinguish elements can be addressed by highlights, depth of field, ...
